### ISSUE TITLE
Mikrotik scan error

### ISSUE THREAD

**Issue by resumeman** [leiweibau/Pi.Alert#425]
I've been using PiAlert for a few weeks, and I find it helpful.

I have a Mikrotik RB5009 router, and every time I edit the config file to allow it, I'm getting errors and it crashes the whole scan. I edited the file to include the IP of the router and my login info. Here's the error message.

Scanning...
    arp-scan Method...
    arp-scan: One interface
    Pi-hole 5 Client List Method... Skipped
    Pi-hole 5 DHCP Leases Method... Skipped
    Fritzbox Method... Skipped
    Mikrotik Method...
Traceback (most recent call last):
File "/opt/pialert/back/pialert.py", line 3974, in
    sys.exit(main())    
             ^^^^^^
File "/opt/pialert/back/pialert.py", line 102, in main
    res = scan_network()
         ^^^^^^^^^^^^^^
File "/opt/pialert/back/pialert.py", line 785, in scan_network
    read_mikrotik_leases()
File "/opt/pialert/back/pialert.py", line 1223, in read_mikrotik_leases
    import routeros_api
ModuleNotFoundError: No module named 'routeros_api'

Do I need to install something separately for it to recognize the router?

**Comment by leiweibau** [comment]
During the setup a special package should be installed

https://github.com/leiweibau/Pi.Alert/blob/2f769280e9a0654c8fed5a657225a1ba9e693275/install/pialert_install.sh#L369

**Comment by resumeman** [comment]
It apparently didn't. Can I install it with that pip3 command retroactively?

**Comment by resumeman** [comment]
BTW, just to be clear, I installed it on a Proxmox LXC using the tteck script, not via docker or some other method

**Comment by resumeman** [comment]
Well I ran the pip command. Got no errors, got no acknowledgement. Went back and re-entered my Mikrotik info in the config file and saved it. Ran the scan and didn't get the error any more, but it was reported that it skipped the MT method...

**Comment by leiweibau** [comment]
If I remember correctly, the root user is used under LXC. Did you also install the package under root? Did the configuration files also activate the import?

**Comment by resumeman** [comment]
So just to provide completeness, the script created the LXC and installed the program. But as I recall, pretty much every Proxmox LXC I've created only had root.  So I presume that is how it's installed.

HOWEVER - It may have just needed some time. After a bit I tried it again and - it no longer skips Mikrotik. So I think I'm all set now. Thanks for pointing me to the package.

Incidentally on the root question - when I started working on the pip install I was in root. pip3 complained about potential problems from running in root, so I created a user and ran it there - and that's where the problem looks to have been solved.
